examples/gallery/embellishments/colorbars_multiple.py

```diff
@@ -0,0 +1,43 @@
+"""
+Multiple colormaps
+------------------
+This gallery shows how to use multiple colormaps for different subplots. To
```

Comment:
```suggestion
This gallery shows how to create multiple colormaps for different subplots. To
```

I think this makes it more clear that there are multiple calls to `makecpt`

Reply:
Great~